CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Add the list of code owners here (using their GitHub username)
 * @alindima
+* @petreeftime
```

Comment:
Can we add Petre on the same line with you (at least for consistency with the other repos)? The github documentation seems to recommend the same thing: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.

Reply:
sure, done